src/backend/package.json

```diff
@@ -21,7 +21,6 @@
     "test:watch": "jest --watch",
     "test:cov": "jest --coverage",
     "test:debug": "node --inspect-brk -r tsconfig-paths/register -r ts-node/register node_modules/.bin/jest --runInBand",
-    "test:e2e": "jest --config ./test/jest-e2e.json",
```

Comment:
I would still keep the script for future reference. when we have enough time and resources to implement it. @akroon3r - if that makes sense?